packages/intent/src/commands/load.ts

```
@@ -1,12 +1,12 @@
-import { fail } from '../cli-error.js'
-import { coreOptionsFromGlobalFlags, printDebugInfo } from '../cli-support.js'
+import { fail } from '../shared/cli-error.js'
+import { coreOptionsFromGlobalFlags, printDebugInfo } from './support.js'
 import {
   IntentCoreError,
   loadIntentSkill,
   resolveIntentSkill,
-} from '../core.js'
-import type { GlobalScanFlags } from '../cli-support.js'
-import type { LoadedIntentSkill, ResolvedIntentSkill } from '../core.js'
+} from '../core/index.js'
```

ESLint

[error] `../core/index.js` import should occur before import of `./support.js`

(import/order)